### PR TITLE
Add language choice - updated logic for DE

### PR DESCRIPTION
**Pull request by bensle**

### CODE REVIEW

File: src/components/Searchbar/Search.js [hunk 1]
```diff
@@ -17,14 +18,27 @@ export default function SearchBar({
   const handleClick = () => {
     refInput.current.focus();
   };
-  //----- if input -> setResult with filterd base array-----//
-  function filteredProduct(input) {
-    let inputReg = new RegExp(input, 'i');
+  //-----Language is EN - if input  -> setResult with filterd base array-----//
+  function filteredProductEN(input) {
+    let inputRegEN = new RegExp(input, 'i');
```
Comment (macfrei) [inline src/components/Searchbar/Search.js:23]: Why use `let` instead of `const`?

File: src/components/Searchbar/Search.js [hunk 2]
```diff
@@ -17,14 +18,27 @@ export default function SearchBar({
...
-      ? setResults(shoppingItems.filter((item) => item.name.en.match(inputReg)))
+      ? setResults(
+          shoppingItems.filter((product) => product.name.en.match(inputRegEN))
+        )
       : setResults([]);
     setSearchInput(input);
   }
+  //----- Language is DE - if input  -> setResult with filterd base array-----//
+  function filteredProductDE(input) {
+    let inputRegDE = new RegExp(input, 'i');
```
Comment (macfrei) [inline src/components/Searchbar/Search.js:33]: Same here.

**Review by macfrei — COMMENTED** [review]
Nice job 👍

File: src/components/Searchbar/Search.js (hunk 1 above)
Comment (bensle) [reply src/components/Searchbar/Search.js:23]: At first i tried to combine EN & DE. i got a console log error which referred to the variable. Didn't really know how to handle that and changed to let - i've also seen this in my little tutorial on regular expressions. Now i changed both to const and it works. Thanks. Im not fully understanding regExp at the moment and it took nearly a whole day for this function to work.

**Pull request merged**